tests/src/Interop/COM/NETClients/Primitives/StringTests.cs

```diff
@@ -191,8 +191,7 @@ private void Marshal_LPWString()
                 Assert.AreEqual(expected, actual);
 
                 actual = local;
-                this.server.Reverse_LPWStr_OutAttr(local, actual); // No-op for strings
-                Assert.AreEqual(local, actual);
+                Assert.Throws<MarshalDirectiveException>( () => this.server.Reverse_LPWStr_OutAttr(local, actual));
```

Comment:
Please also update the managed implementation of `Reverse_LPWStr_OutAttr()` to throw an exception. Or at least documented as desired.